internal/config/cgroupmanager/cgroupmanager.go

```diff
@@ -0,0 +1,69 @@
+// +build linux
+
+package cgroupmanager
+
+import (
+	"github.com/pkg/errors"
+)
+
+const (
+	cgroupMemorySubsystemMountPathV1 = "/sys/fs/cgroup/memory"
+	cgroupMemorySubsystemMountPathV2 = "/sys/fs/cgroup"
+	scopePrefix                      = "crio"
+	// minMemoryLimit is the minimum memory that must be set for a container.
+	// A lower value would result in the container failing to start.
+	minMemoryLimit = 12582912
```

Comment:
1. Maybe add a comment that this is an empirical value valid for x86_64.

2. Maybe put it as `12 * 1024 * 1024` for readability.